tests/cpp/block_manager.cpp

```diff
@@ -4,13 +4,13 @@
 
 #include <gtest/gtest.h>
 #include "openvino/runtime/core.hpp"
-#include "continuous_batching_pipeline.hpp"
-#include "sequence_group.hpp"
-#include "scheduler.hpp"
+#include "openvino/genai/continuous_batching_pipeline.hpp"
 #include "openvino/genai/generation_config.hpp"
+#include "../../src/cpp/src/sequence_group.hpp"
```

Comment:
Please do not  use relative paths?

Reply:
I don't see a better way to implement it. `sequence_group.hpp` was private and I preserved that. But the tests need the declaration. I could paste the `sequence_group.hpp`'s content here, but it feels to be even worse.

Reply:
There is different approach not to use relative paths :) For example, include sources to the cmake tests directly by [`target_include_directories`](https://cmake.org/cmake/help/latest/command/target_include_directories.html)